R/compare.R

```diff
@@ -140,10 +140,10 @@ order_proxy <- function(proxy, direction = "asc", na_value = "largest") {
       return(integer(0L))
     }
     args <- map(unname(proxy), function(.x) {
-      if (is.data.frame(.x))
-        vec_order(.x, direction = direction, na_value = na_value)
-      else
-        .x
+      if (is.data.frame(.x)) {
+        .x <- vec_order(.x, direction = direction, na_value = na_value)
+      }
+      .x
```

Comment:
There's an `else` missing right?

Reply:
it used to be `if(is.data.frame(.x)) vec_order(.x, ...) else .x` now `.x` get reassigned and then we always return `.x` I can rollback to previous version but it felt weird to have `{ .x }`